### PR TITLE
Add tests for calling Document.prototype.adoptNode and importNode on a shadow root

### PR DESCRIPTION
**Pull request by rniwa**

### CODE REVIEW

File: shadow-dom/Document-prototype-adoptNode.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>DOM and Shadow DOM: Document.prototype.adoptNode</title>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<meta name="assert" content="The adoptNode(node) method must throw a HierarchyRequestError exception if node is a shadow root.">
+<link rel="help" href="https://dom.spec.whatwg.org/#dom-document-adoptnode">
+<script src="/testharness.js"></script>
+<script src="/testharnessreport.js"></script>
```
Comment (TakayoshiKochi) [inline shadow-dom/Document-prototype-adoptNode.html]: Path to testharness.js/testharnessreport.js should be /resources/*.js.

File: shadow-dom/Document-prototype-importNode.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>DOM and Shadow DOM: Document.prototype.importNode</title>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<meta name="assert" content="The importNode(node, deep) method must throw a NotSupportedError exception if node is a shadow root.">
+<link rel="help" href="https://dom.spec.whatwg.org/#dom-document-importnode">
+<script src="/testharness.js"></script>
+<script src="/testharnessreport.js"></script>
```
Comment (TakayoshiKochi) [inline shadow-dom/Document-prototype-importNode.html]: Path to testharness.js/testharnessreport.js should be /resources/*.js.

**Comment by rniwa** [comment]
@annevk @hayatoito @travisleithead @inscriber: these are tests for `importNode` and `adoptNode`

**Comment by annevk** [comment]
These should probably just be in the dom/ directory, no? Also, \o/

**Pull request merged**